rfc/java11.md

```diff
@@ -0,0 +1,173 @@
+# Java 11 Migration
+
+| | |
+|-|-|
+| **Status**     | _**Proposed**, Accepted, Implemented, Obsolete_ |
+| **RFC #**      | [62](https://github.com/spinnaker/community/pull/62) |
+| **Author(s)**  | Michael Plump ([`@plumpy`](https://github.com/plumpy)) |
+| **SIG / WG**   | Platform SIG |
+
+## Overview
+
+To keep current with the evolution of Java, Spinnaker needs to move from Java 8
+to Java 11. This RFC documents a process and timeline for doing so.
+
+### Goals and Non-Goals
+
+Goals:
+
+*   Provide a process for moving official Spinnaker builds to the Java 11 JRE
+*   Discuss ways we will encourage people using non-offical builds to also move
+    to the Java 11 JRE
+*   Provide a timeline for the final switch to a Java 11 JDK along with criteria
+    to decide if the timeline should be delayed
+
+Non-Goals:
+
+*   Providing a rationale for upgrading Java; all active Java projects need to
+    update their JDK version eventually as libraries we depend upon will also
+    depend on new versions of the JDK
```

Comment:
I think this is a good point, ideally everything would upgrade at the same time (same release window) as maintaining multiple services under different versions would not be ideal.